ui/component/commentCreate/tip-contents.jsx

```diff
@@ -63,7 +63,7 @@ export const TipActionButton = (tipButtonProps: TipButtonProps) => {
       <Button
         {...buttonProps}
         title={name}
-        label={tipSelectorOpen ? __('Switch to %tip_method%', { tip_method: name }) : undefined}
+        label={tipSelectorOpen ? __('%tip_method%', { tip_method: name }) : undefined}
```

Comment:
This one's minor, but want to avoid it from being auto-generated in app-string later.

Change it to `? name : undefined}` since it's no longer a sentence, and `name` has be wrapped in __() before passed in.